workbase/src/renderer/components/SchemaDesign/store/actions.js

```diff
@@ -253,30 +251,29 @@ export default {
 
     await state.schemaHandler.deleteAttribute(payload);
 
-    await dispatch(COMMIT_TX, graknTx).then(async () => {
-      const node = state.visFacade.getNode(state.selectedNodes[0].id);
-      node.attributes = Object.values(node.attributes).sort((a, b) => ((a.type > b.type) ? 1 : -1));
-      node.attributes.splice(payload.index, 1);
-      state.visFacade.updateNode(node);
-
-      // delete edge to attribute type
-      graknTx = await dispatch(OPEN_GRAKN_TX);
-
-      const attributeTypeId = (await graknTx.getSchemaConcept(payload.attributeLabel)).id;
-      const edgesIds = state.visFacade.edgesConnectedToNode(state.selectedNodes[0].id);
-      edgesIds.forEach((edgeId) => {
-        const edge = state.visFacade.getEdge(edgeId);
-        if (edge.to === attributeTypeId) {
-          state.visFacade.deleteEdge(edgeId);
-        }
-      });
-      graknTx.close();
-    })
+    await dispatch(COMMIT_TX, graknTx)
```

Comment:
I think we should remove the:

```
if (await (await type.instances()).next()) throw Error('Cannot remove attribute type from schema concept with instances.');
```

It is not Workbase duty to check the correctness of the graph, the server should return an exception if we try to delete types that have instances, the workbase duty is to correctly handle those exceptions and show them in a fancy way